src/styles/block/stars.scss

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
Combine with `background` property (with background-image)

Reply:
Try to do it